cfgov/cfgov/settings/base.py

```diff
@@ -230,12 +230,15 @@
 if ALLOW_ADMIN_URL:
     DATA_UPLOAD_MAX_NUMBER_FIELDS = 2000  # For heavy Wagtail pages
 
-# Databases
-DATABASES = {}
-
-# If DATABASE_URL is defined in the environment, use it to set the Django DB
-if os.getenv("DATABASE_URL"):
-    DATABASES["default"] = dj_database_url.config()
```

Comment:
Prior to this change, it was at least theoretically possible to pass this code without having any databases defined. I can't think of any cases where we actually depend on this logic, though.